# Plant Breeder & Data analyst

#### Summary: 
Ph.D. in Plant Breeding and Genetics with 6 years of research experience in genetics, genomics, and data analysis of crop plants. Skilled in experimental design, statistical analysis, handling large genomic data and development and deployment of R/Shiny packages for plant breeding applications. Interested in a research or industry role involving genetics, plant breeding, bioinformatics, or Artificial Intelligence for plant breeding. 
#### Publication Statistics:
Cumulative Impact Factor: **34.171**

Total Citations: **39** <a href="https://scholar.google.com/citations?user=KnuQm0cAAAAJ&hl=en" target="_blank">Google Scholar</a>

#### Technical Skills:

- Programming Languages: _Proficient in R, Python, Linux command line; experience with Shiny, Google Colab._
- Phenomics and Genomics: _Expertise in molecular breeding techniques, NGS data analysis, QTL/GWAS, genomic selection, designing field trials, managing phenotyping pipelines, and analyzing high-throughput phenotypic and genotypic data._
- Data Science: _Multivariate analysis, machine learning, large scale genomic data analysis and visualization._
  
## üéì Education
- **Ph.D.**, Genetics and Plant Breeding | <a href="https://tnau.ac.in/" target="_blank">TamilNadu Agricultural University</a> (_FEB 2021 - Thesis submitted on 31st OCT 2023_)		  		
- **M.Sc.**, Genetics and Plant Breeding	| <a href="https://tnau.ac.in/" target="_blank">TamilNadu Agricultural University</a> (_AUG 2017 - DEC 2020_)	 			        		
- **B.Sc.**, Agriculture | <a href="https://cattheni.edu.in/" target="_blank">College of Agricultural Technology</a> (_JUL 2013 - JUL 2017_)

## üî¨ Research Experience
**Ph.D. research @<a href="https://tnau.ac.in/" target="_blank">TNAU</a> (_2021 - 2023_)**
- Developed and applied advanced tissue culture techniques like embryo culture to rapidly generate **somaclonal variants of rice** with improved salinity stress tolerance.
- Identified and validated novel genomic regions **(QTLs)** governing salinity tolerance through **molecular mapping** in rice F<sub>2</sub> populations along with thorough phenotypic and biochemical characterization.
- Employed **multi-omics approaches** including GC-MS metabolomics to uncover key genes and metabolites linked to superior osmotic/ionic adjustment under salt stress.

**Research scholar @ <a href="https://www.icrisat.org/" target="_blank">ICRISAT</a> (_2018 - 2020_)**
- Single plant phenotying of diverse germplasm accessions (Sorghum, Peral millet, Pigeon pea ‚Äì genotyped 1,980 plants/samples combined) to examine intra and inter accession genetic diversity. 
- Single plant genotyping of accessions using **<a href="https://www.diversityarrays.com/" target="_blank">DArTSeq</a> based SNPs**.
- Harnessed whole genome genotyping, simulations, and predictive modeling to provide integrated insights into the dynamics of genetic variation across a species' range, by quantifying genomic diversity within and between isolated subpopulations, and developed optimized statistical frameworks to guide sustainable sampling regimes that limit genetic drift.

## üíº Work experience 
**Data Science Consultant @ <a href="https://www.fiverr.com/allanvictor_?public_mode=true" target="_blank">Fiverr</a> (_Decembe 2020 - Present_)**
- 5‚≠ê rated freelancer with over 200 hours of data analysis and visualization projects completed in R with over six years of experience specializing in genomic data analysis and bioinformatics.
- Proficient in handling big data and performing complex modelling using popular R libraries and packages such as tidyr, data.table, dplyr, plyr, tenserflow, ggplot, ggdendro, ggtree, ggheatmap, and circos. 
- Received positive feedback from clients for knowledge, professionalism, and mastery of R programming. Demonstrated ability to deliver high-quality work as evidenced by the portfolio available on the <a href="https://www.fiverr.com/allanvictor_?public_mode=true" target="_blank">Profile</a>. 

## üíª Programming and data analysis skills
- Proficient in full stack development of **R packages** using modular coding practices.
- Created **production-grade Shiny web applications** for interactive data analysis and visualization and; expertise in dependency management tools like Golem for scalable deployment.
- **Machine learning model** building for image classification/segmentation tasks; trained CNNs and other deep learning architectures in R, Python (PyCharm).
- **Multivariate data analysis** of large-scale omics datasets including genomics, phenomics and metabolomics using cutting-edge bioinformatics tools.
- Experience with analysis of **next-generation sequencing** data including quality control, read mapping, variant calling, expression quantification, metagenomic profiling, and associated statistical analysis using standard workflows in R and Python.
- **Advanced visualization** for multi-dimensional biological data through Circos, ggtree, ggtreeextra, Cytoscape and other platforms.

**Additional Skills:**
  
- Git/GitHub for version control and collaborative coding.
- High performance computing on clusters for scalable data analysis.
- Bioconductor for analyzing genomic/transcriptomic experiments.
- Workflow automation to enhance reproducibility and to faster workflow. 

## üå± R Packages developed
### ‚úÖ PBGeno
<a href="https://github.com/Allan-gitrepos/pbgeno" target="_blank">GitHub</a>

Developed pbgeno, an R package to streamline data analysis workflows for plant breeders. The package provides functions for calculating genetic distances, clustering genotypes, estimating diversity statistics, creating publication-quality visualizations, and automating routine tasks. Key features include calculating genetic distance matrices, structure-based clustering, polymorphism quantification, and converting proprietary marker genotypes into standardized formats for genome-wide association mapping. 

### ‚úÖ PBPerfect
**PBPerfect** <a href="https://allanbiotools.shinyapps.io/pbperfect/" target="_blank">(Visit Page)</a> is a interactive web tool enabling reproducible multivariate analysis with visualization of phenotypic and genotypic data. It features basic statistics, experimental designs, SSR workflows, multivariate analysis, mating designs, and dynamic graphics with outputs exported as publication-standard tables and graphics requiring no further formatting.

 Here is a draft GitHub readme section for your PBlinkagemap package:

### ‚úÖ PBlinkagemap

**PBlinkagemap** <a href="https://allanbiotools1.shinyapps.io/PBLinkagemap/" target="_blank">(Visit Page)</a> enables easy creation of linkage maps and identification of associated quantitative trait loci (QTLs) from genomic and phenotypic datasets. 

It allows users to:
- Import chromosome, marker, map distance and trait score data
- Interactively explore results on linkage maps
- Visualize QTL locations and effects
By handling computationally intensive linkage analysis and mapping behind the scenes, PBlinkagemap makes it simple for users to go from datasets to QTL discovery through an intuitive interface.

### ‚úÖ PB-GWAS üß¨

**PB-GWAS** <a href="https://allanbiotools1.shinyapps.io/PBgwas/" target="_blank">(Visit Page)</a> makes powerful genome-wide association studies accessible through an easy-to-use web app üë©‚Äçüíª

Key features:
üì• File upload in 4 clicks  
üèÉ‚Äç‚ôÇÔ∏è One-click GWAS launch  
‚öôÔ∏è Adjust parameters via sidebar   
üìà Interactive result plotting  
üìÑ Full PDF report downloading

By eliminating coding barriers, PB-GWAS allows both new and advanced users to leverage GAPIT workflows with no programming expertise required!. Whether you want to map simple or complex traits, PB-GWAS provides the automated analysis to accelerate discoveries üî¨

 Here is a draft GitHub readme for the PBHaploMineR package:

### ‚úÖ PBHaploMineR üß¨

**PBHaploMineR** <a href="https://github.com/Allan-gitrepos/PBhaplominerR" target="_blank">(Visit Page)</a> provides a toolkit to streamline pangenome haplotype mining and comparison from next-generation sequencing data. This R package aims to make large-scale haplotype analysis efficient and accessible for species with reference pangenomes.

**Key Features:**
* **Sequence Import** - Functions to import raw reads from multiple platforms and store in standardized schema
* **Haplotype Calling** - Optimized algorithms for pangenome-wide haplotype calling, incorporating structural variation
* **HapViz** - Interactive visualization system to explore and compare haplotypes in context of pangenome structure
* **HapCompare** - Statistically compare haplotypes between groups of samples/accessions and identify associated genomic signatures
* **Parallelization** - Built-in parallelization to scale analyses across HPC infrastructure

PBHaploMineR is still under development and testing. ETA for first stable version is Q1 2024.

## üé§ Workshop and Conferences
- Attended the workshop on "<a href="https://twitter.com/GenebankICRISAT/status/1169137453472669696" target="_blank">A15 DArTSeq Data Analysis</a>" at <a href="https://www.cimmyt.org/" target="_blank">CIMMYT</a>, Mexico
- Attended the international conference on <a href="https://twitter.com/GenebankICRISAT/status/1158297753371308032" target="_blank">"Neglected and Underutilized Crop Species for Food, Nutrition, Energy and Environment"</a> at NIPGR, New Delhi. Awarded a travel grant in recognition of contribution. 
- Attended the workshop on **<a href="https://www.facebook.com/media/set/?vanity=natesan.senthil&set=a.5804435956243670" target="_blank">HPLC: "Principles and Applications in Plant Metabolomics"</a>**" at TNAU,Coimbaotore.
- Attended the workshop on **<a href="https://www.covaichronicle.com/english/contentview/natu?fbclid=IwAR0fA8w7ubx__cGaKnmE-68-tmRLg92XHs1Rf-HnhQzRjmFc_calVar8yFo" target="_blank">Molecular Modelling and Docking</a>** at TNAU, Coimbatore.
- Presented poster titled "**Genes for salt tolerance revealed by functional metagenomics in rice**" at 6<sup>th</sup> National conference AC&RI, TNAU, Trichy.

## ‚úçÔ∏è Articles & Blogs
- <a href="https://medium.com/@victorallan" target="_blank">Medium Articles</a>

## üìú Publications

1. Allan Victor, Mani Vetriventhan, Ramachandran Senthil, S. Geetha, Santosh Deshpande, Abhishek Rathore, Vinod Kumar, Prabhat Singh, Surender Reddymalla, and V√¢nia CR Azevedo. **"Genome-wide DArTSeq genotyping and phenotypic based assessment of within and among accessions diversity and effective sample size in the diverse sorghum, pearl millet, and pigeonpea landraces."**_Frontiers in Plant Science_ 11 (2020): 587426. (DOI: <a href="https://doi.org/10.3389/fpls.2020.587426" target="_blank">https://doi.org/10.3389/fpls.2020.587426</a>)
2. Allan, V. (2023) **‚ÄòPB-Perfect: A Comprehensive R-Based Tool for Plant Breeding Data Analysis‚Äô**, PB - Perfect. Available at: <a href="https://allanbiotools.shinyapps.io/pbperfect/" target="_blank">https://allanbiotools.shinyapps.io/pbperfect/</a>.
3. Backiyalakshmi, C., Mani Vetriventhan, Santosh Deshpande, C. Babu, Allan Victor, D. Naresh, Rajeev Gupta, and Vania CR Azevedo. **"Genome-wide assessment of population structure and genetic diversity of the global finger millet germplasm panel conserved at the ICRISAT Genebank."** _Frontiers in Plant Science_ 12 (2021): 692463. (DOI: <a href="https://doi.org/10.3389/fpls.2021.692463" target="_blank">https://doi.org/10.3389/fpls.2021.692463</a>)
4. Vetriventhan, Mani, Hari D. Upadhyaya, Vania CR Azevedo, Allan Victor, and Seetha Anitha. **"Variability and trait‚Äêspecific accessions for grain yield and nutritional traits in germplasm of little millet (_Panicum sumatrense_ Roth. Ex. Roem. & Schult.)."** _Crop Science_ 61, no. 4 (2021): 2658-2679. (DOI: <a href="https://doi.org/10.1002/csc2.20527" target="_blank">https://doi.org/10.1002/csc2.20527</a>)
5. Jagadesh, M., Duraisamy Selvi, Subramanium Thiyageshwari, Thangavel Kalaiselvi, Allan Victor, Munmun Dash, Keisar Lourdusamy, Ramalingam Kumaraperumal, Pushpanathan Raja, and U. Surendran. **"Exploration of microbial signature and carbon footprints of the Nilgiri Hill Region in the Western Ghats global biodiversity hotspot of India."** _Applied Soil Ecology_ (2023): 105176 (DOI: <a href="https://doi.org/10.1016/j.apsoil.2023.105176" target="_blank">https://doi.org/10.1016/j.apsoil.2023.105176</a>).
6. Jagadesh, M., Cherukumalli Srinivasarao, Duraisamy Selvi, Subramanium Thiyageshwari, Thangavel Kalaiselvi, Aradhna Kumari, Santhosh Kumar Singh, Allan Victor **"Quantifying the Unvoiced Carbon Pools of the Nilgiri Hill Region in the Western Ghats Global Biodiversity Hotspot‚ÄîFirst Report."** _Sustainability_ 15, no. 6 (2023): 5520. (DOI: <a href="https://doi.org/10.3390/su15065520" target="_blank">https://doi.org/10.3390/su15065520</a>)
7. Jagadesh, M., Duraisamy Selvi, Subramanium Thiyageshwari, Cherukumalli Srinivasarao, Thangavel Kalaiselvi, Keisar Lourdusamy, Ramalingam Kumaraperumal, and Victor Allan. **"Soil Carbon Dynamics Under Different Ecosystems of Ooty Region in the Western Ghats Biodiversity Hotspot of India."** _Journal of Soil Science and Plant Nutrition_ 23, no. 1 (2023): 1374-1385. (DOI: <a href="https://doi.org/10.1007/s42729-023-01129-2" target="_blank">https://doi.org/10.1007/s42729-023-01129-2</a>)
8. Allan Victor., N. Meenakshi Ganesan, R. Saraswathi, R. Gnanam, and C. N. Chandrasekhar. **"Exploring the phenotypic diversity of rice: A multivariate analysis of local landraces and elite cultivars of Tamil Nadu and Exotic Lines."** _Electronic Journal of Plant Breeding_ 14, no. 3 (2023): 857-866. (DOI: <a href="https://ejplantbreeding.org/index.php/EJPB/article/view/4804" target="_blank">10.37992/2023.1403.099</a>)
9. Allan Victor, S. Geetha, Mani Vetriventhan, and V√¢nia CR Azevedo. **"Genetic diversity analysis of geographically diverse landraces and wild accessions in sorghum."** _Electronic Journal of Plant Breeding_ 11, no. 03 (2020): 760-764. (DOI: <a href="https://www.ejplantbreeding.org/index.php/EJPB/article/view/3575" target="_blank">https://doi.org/10.37992/2020.1103.125</a>)
    

## üìö References

|     |     |
|:----|:----|
| **Name:** | **Dr. Vania de Azevedo** |
| **Position:** | Former Head, Plant Genetic Resources |
| **Organization:** | ICRISAT, Hyderabad, India |
| **E-mail:** | <a href="mailto:azevedovcr@gmail.com">azevedovcr@gmail.com</a> |
| **LinkedIn:** | <a href="https://www.linkedin.com/in/v%C3%A2nia-c-renn%C3%B3-azevedo-21493449/" target="_blank">Visit Page</a> |

|     |     |
|:----|:----|
| **Name:** | **Dr. Mani Vetriventhan** |
| **Position:** | Senior Scientist, Plant Genetic Resources |
| **Organization:** | ICRISAT, Hyderabad, India |
| **E-mail:** | <a href="mailto:M.Vetriventhan@cgiar.org">M.Vetriventhan@cgiar.org</a> |
| **LinkedIn:** | <a href="https://www.linkedin.com/in/vetriventhan-mani-26468322/" target="_blank">Visit Page</a> |

|     |     |
|:----|:----|
| **Name:** | **Mr. Rajaguru Bohar** |
| **Position:** | Regional Genotyping Coordinator (South Asia) / Senior Scientist (Project management) |
| **Organization:** | CIMMYT |
| **E-mail:** | <a href="mailto:wishmeguru@gmail.com">wishmeguru@gmail.com</a> |
| **LinkedIn:** | <a href="https://www.linkedin.com/in/rajaguru-bohar-5038762a/" target="_blank">Visit Page</a> |
|     |     |


## üìû Contact

| **Name**   | Allan. V | 
| **E-mail**   | <a href="mailto:albertoogy@gmail.com">albertoogy@gmail.com</a> | 
| **LinkedIn**   | <a href="https://www.linkedin.com/in/allan-victor-6919b4152/" target="_blank">Visit Page</a> | 
